docs/enterprise-guide/dashboard-subscriptions.md

```diff
@@ -0,0 +1,28 @@
+
+## Customize filter values for each dashboard subscription
+
+This page covers the Enterprise Edition features for dashboard subscriptions. Our User's Guide can give you an introduction to [dashboard subscriptions](../users-guide/dashboard-subscriptions.md). 
+
+Enterprise Edition lets you set up multiple subscriptions for a single dashboard, and lets you customize which filter values to apply to each subscription. That way you can send one group of people an email (or Slack message) the contents of the dashboard with one filter value applied, and set up another subscription to send the dashboard's contents with a different filter applied. You only need to maintain one dashboard, and you can set up different subscriptions to give different groups of people the info they need. 
+
+### Adding a subscription
```

Comment:
I think this should already be covered in the main dashboard subscriptions doc, no?